src/System Application/App/Permissions/SystemApplicationEdit.PermissionSet.al

```diff
@@ -29,5 +29,6 @@ permissionset 22 "System Application - Edit"
                              "Language - Edit",
                              "PageScripting - Rec",
                              "Translation - Edit",
-                             "Word Templates - Edit";
+                             "Word Templates - Edit",
+                             "Data Analysis - Edit";
```

Comment:
Once the permission has been moved to "Data Analysis - Exec", replace this with "Data Analysis - Exec".